pkg/controller/atlasmap/atlasmap_controller.go

```diff
@@ -32,7 +32,9 @@ func Add(mgr manager.Manager) error {
 
 // newReconciler returns a new reconcile.Reconciler
 func newReconciler(mgr manager.Manager) reconcile.Reconciler {
-	return &ReconcileAtlasMap{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	r := &ReconcileAtlasMap{client: mgr.GetClient(), scheme: mgr.GetScheme()}
+	r.createConsoleYAMLSamples()
```

Comment:
We only want to do this on OpenShift versions that support `ConsoleYAMLSamples`.